node/src/accountant/scanners/payable_scanner/tx_templates/priced/new.rs

```diff
@@ -86,7 +86,7 @@ impl PricedNewTxTemplates {
         ceil: u128,
     ) -> String {
         format!(
-            "The computed gas price {} wei is above the ceil value of {} wei set by the Node.\n\
+            "The computed gas price {} wei is above the ceil value of {} wei computed by this Node.\n\
```

Comment:
The word "ceil" is a nonsense. And I don't think this kind of shortcut is widely used.

Reply:
This is a commonly used word in programming terminology. See:

1. [Example 1 - JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil)
2. [Example 2 - C](https://www.geeksforgeeks.org/c/c-ceil-function/)
3. [Example 3 - Python](https://docs.python.org/3.9/library/math.html?highlight=math%20ceil#math.ceil)
4. [Example 4 - Rust](https://docs.rs/libmath/latest/math/round/fn.ceil.html)

I suggest you get used to it, it's short and it conveys what we mean.